src/Crawler.php

```diff
@@ -74,6 +81,18 @@ public function setConcurrency(int $concurrency)
         return $this;
     }
 
+	/**
+	 * @param int $depth
+	 *
+	 * @return $this
+	 */
+	public function setDepth(int $depth)
```

Comment:
Could you rename this function to `setMaximumDepth` and all variables `$depth` to `maximumDepth`?

Reply:
Absolutely. This makes more sense.